pwa/src/apiService/resources/endpoint.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosInstance, AxiosResponse } from "axios"
```

Comment:
This file does not have consistent whitespace (we're expecting one white line between all functions).